### PR TITLE
R2 release notes fixes

### PR DESCRIPTION
**Pull request by frosty-geek**
* fixed openstack-health-monitor url
* fixed typos & formating

Signed-off-by: Ralf Heiringhoff <ralf.heiringhoff@plusserver.com>

### CODE REVIEW

File: Release-Notes/Release2.md [hunk 1]
```diff
@@ -168,15 +168,15 @@ trademark certification.
...
 Our partner PlusServer has [achieved](https://www.openstack.org/brand/interop/)
 a [BSI C5](https://www.bsi.bund.de/EN/Topics/CloudComputing/Compliance_Criteria_Catalogue/Compliance_Criteria_Catalogue_node.html)
-security certification for their SCS implementation PlusCloud Open.
+security certification for their SCS implementation pluscloud open.
 
 We are working within [Gaia-X](https://gaia-x.eu/) to further the power
 of Gaia-X self-descriptions and are closely working with the
 [GXFS project](https://gxfs.de/)
-to jointly deliver a standard toolbox for Gaia-X conformant
+to jointly deliver a standard toolbox for Gaia-X conformance
```
Comment (itrich) [inline Release-Notes/Release2.md:176]: Maybe we should write `Gaia-X compliant infrastructure` ?

**Review by itrich — CHANGES_REQUESTED** [review]
Thank you for proof-reading the release notes and all your contributed improvements!

File: Release-Notes/Release2.md (hunk 1 above)
Comment (fkr) [reply Release-Notes/Release2.md:176]: yes, good one.

File: Release-Notes/Release2.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ allowing for long-living management nodes and workload clusters managed by them.
 the same name in the manager environment (manager-configuration.yml). All configuration
 parameters from environments/configuration.yml should be moved to environments/manager/configuration.yml.
 
-* In kolla-ansible the haproxy role was renamed to loadbalancer. Accordingly, loadbalancer must now be
+* In kolla-ansible the HAProxy role was renamed to load balancer. Accordingly, load balancer must now be
```
Comment (fkr) [inline Release-Notes/Release2.md:119]: The original changelog from kolla-ansible states:

"A new group loadbalancer is required in inventory file prior to upgrade. The loadbalancer group is a replacement for the haproxy group."

I'd like to stick with the upstream spelling.

**Review by fkr — CHANGES_REQUESTED** [review]

File: Release-Notes/Release2.md (hunk 2 above)
Comment (frosty-geek) [reply Release-Notes/Release2.md:119]: aspell disagree's but I don't mind it either way... there were several typos and I only made sure that the spelling was consistent in the whole document.

Comment (fkr) [reply Release-Notes/Release2.md:119]: I agree and in certain places I also put 'HAProxy' (which is the proper spelling for the project), in this case the role is named 'haproxy' and 'loadbalancer' (eg. refering to the technical role names).

Comment (frosty-geek) [reply Release-Notes/Release2.md:119]: As a non-native speaker I heavly rely on aspell or web resources see https://www.spellchecker.net/load%20balancer

File: Release-Notes/Release2.md [hunk 3]
```diff
@@ -116,7 +116,7 @@ allowing for long-living management nodes and workload clusters managed by them.
 the same name in the manager environment (manager-configuration.yml). All configuration
 parameters from environments/configuration.yml should be moved to environments/manager/configuration.yml.
 
-* In kolla-ansible the haproxy role was renamed to loadbalancer. Accordingly, loadbalancer must now be
+* In kolla-ansible the HAProxy role was renamed to loadbalancer. Accordingly, loadbalancer must now be
```
Comment (fkr) [inline Release-Notes/Release2.md:119]: Sorry to be picky about this: haproxy (that is - according to this link: https://github.com/openstack/kolla-ansible/blob/master/releasenotes/notes/drop-group-haproxy-6119957627811873.yaml)

**Review by itrich — APPROVED** [review]
Looks good to me now!

**Comment by fkr** [comment]
thanks @frosty-geek

**Pull request merged**